agent/engine/engine_integ_test.go

```diff
@@ -169,6 +169,11 @@ func TestHostVolumeMount(t *testing.T) {
 	data, err := ioutil.ReadFile(filepath.Join(tmpPath, "hello-from-container"))
 	assert.Nil(t, err, "Unexpected error")
 	assert.Equal(t, "hi", strings.TrimSpace(string(data)), "Incorrect file contents")
+
+	client := taskEngine.(*DockerTaskEngine).client
+	for _, aVolume := range testTask.Volumes {
+		client.RemoveVolume(context.TODO(), aVolume.Name, 5*time.Second)
```

Comment:
You can create a constant for those `5*time.Second`, or just use default remove volume timeout.